tethys_cli/scaffold_commands.py

```diff
@@ -340,6 +340,7 @@ def scaffold_command(args):
         "project": project_name,
         "project_dir": project_dir,
         "project_url": project_name.replace("_", "-"),
+        "prefix_url": settings.PREFIX_URL.strip("/"),
```

Comment:
It's not a good idea to pass the PREFIX_URL to the scaffold command. Scaffolding is a development activity, while PREFIX_URL is usually handled in production deployment. Please revert this.